liquidctl/driver/smart_device.py

```diff
@@ -643,7 +643,7 @@ def _write_colors(self, cid, mode, colors, sval, direction='forward',):
             for [g, r, b] in colors:
                 self._write([0x22, 0x10, cid])  # clear out all independent LEDs
                 self._write([0x22, 0x11, cid])  # clear out all independent LEDs
-                color_lists = [] * 3
+                color_lists = [0] * 3
```

Comment:
Looking back into how the broken line of coded ended up here (mea culpa), it seems that the intention was to create a list of *four* empty lists:

```suggestion
                color_lists = [[]] * 4
```

The first three lists would be filled in, but the forth would be left empty.